cmd/nodeJsBuild_test.go

```diff
@@ -0,0 +1,140 @@
+package cmd
+
+import (
+	"github.com/SAP/jenkins-library/pkg/mock"
+	"github.com/bmatcuk/doublestar"
+	"github.com/stretchr/testify/assert"
+	"sort"
+	"testing"
+)
+
+type nodeJsBuildMockUtilsBundle struct {
+	execRunner mock.ExecMockRunner
+	files      map[string][]byte
+}
+
+func (u *nodeJsBuildMockUtilsBundle) fileExists(path string) (bool, error) {
+	_, exists := u.files[path]
+	return exists, nil
+}
+
+// duplicated from nexusUpload_test.go for now, refactor later?
+func (u *nodeJsBuildMockUtilsBundle) glob(pattern string) ([]string, error) {
+	var matches []string
+	for path := range u.files {
+		matched, _ := doublestar.Match(pattern, path)
+		if matched {
+			matches = append(matches, path)
+		}
+	}
+	// The order in m.files is not deterministic, this would result in flaky tests.
+	sort.Sort(byLen(matches))
+	return matches, nil
+}
+
+func (u *nodeJsBuildMockUtilsBundle) getwd() (dir string, err error) {
+	return "/project", nil
+}
+
+func (u *nodeJsBuildMockUtilsBundle) dir(fileName string) string {
+	return "/project"
+}
+
+func (u *nodeJsBuildMockUtilsBundle) chdir(dir string) error {
+	return nil
+}
```

Comment:
Maybe it is not worth it, but you are not actually testing that the code runs npm in the expected sub-folders.